x-pack/plugins/security_solution/public/resolver/view/process_event_dot.tsx

```diff
@@ -453,13 +451,13 @@ export const ProcessEventDot = styled(UnstyledProcessEventDot)`
     stroke-dashoffset: 500;
     fill-opacity: 0;
   }
-  &:hover:not([aria-current]) .backing {
+  &:hover:not([aria-selected]) .backing {
```

Comment:
@bkimmel can you chime in on this?

Reply:
I think maybe we want to leave this how it is for now, which I think we can do if we set `activeDescendant` above.